.Rbuildignore

```diff
@@ -1,3 +1,4 @@
+^data-raw$
```

Comment:
@jwildfire - Since we've added data-raw folder to the buildignore, I'm assuming the tests can't access it and Travis fails.  Can you use the .rda file instead?